game/views.py

```
@@ -1016,9 +1016,17 @@ def post(self, request, *args, **kwargs):
         if limited_response:
             return limited_response
 
-        response = self.form_valid(form)
-        self._record_password_reset_request(request)
-        return response
+        try:
+            response = self.form_valid(form)
+            self._record_password_reset_request(request)
+            return response
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)